app/ui/src/app/connections/common/configuration/validation.component.html

```diff
@@ -1,6 +1,6 @@
 <div [ngClass]="{'text-right': placement === 'right'}">
   <!-- validate button -->
-  <button *ngIf="showValidateButton(connection?.connector?.id)"
+  <button *ngIf="showValidateButton(connection?.connector)"
```

Comment:
Just wanted to point out something for the future: DO NOT introduce function calls in template evaluation expressions such as [ngIf], [hidden] and the like. Otherwise those functions will be evaluated on every pass of the change detection loop. If the function features a heavy execution, well wind up consuming lotsa resources unnecessarily.

In this case the check required is a one time check so it should have been handled within the OnInit() lifecycle handler.